lint/deferInLoop_checker.go

```diff
@@ -0,0 +1,51 @@
+package lint
+
+//! Detects defer in loop and warns that it will not be executed till the end of function's scope.
+//
+// @Before:
+// for i := range [10]int{} {
+// 		defer f(i) // will be executed only at the end of func
+// }
+//
+// @After:
+// for i := range [10]int{} {
+// 		func() {
+// 			defer f(i)
+//	 	}
```

Comment:
Maybe this inner func should accept `i` argument and actually be a call?
Like:
```go
func (i int) { defer f(i) } (i)
```
Otherwise it looks not right.